lib/waterline/query/dql/update.js

```diff
@@ -114,21 +114,7 @@ function createBelongsTo(valuesObject, cb) {
     if(!modelName) return next();
 
     var model = self.waterline.collections[modelName];
-    var pkValue = valuesObject.originalValues[item][model.primaryKey];
-
-    var criteria = {};
-
-    var pkField = hop(model._transformer._transformations, model.primaryKey) ? model._transformer._transformations[model.primaryKey] : model.primaryKey;
-
-    criteria[pkField] = pkValue;
-
-    // If a pkValue if found, do a findOrCreate and look for a record matching the pk.
-    var query;
-    if(pkValue) {
-      query = model.findOrCreate(criteria, valuesObject.values[item]);
-    } else {
-      query = model.create(valuesObject.values[item]);
-    }
+    var query = model.create(valuesObject.values[item]);
```

Comment:
As far as I can tell it was/is a way to do like a nested update. so you would update the table and then also as part of your update do like 

```javascript
.update({email: 'foo@bar.com', relatedTable: {'id': 'dec_123'}) 
```

and then Waterline would find or create the related record for you.

I don't think that we want this feature